test/tests/vectorpostprocessors/csv_reader/transfer/master.i

```diff
@@ -0,0 +1,71 @@
+[Mesh]
+  type = GeneratedMesh
+  parallel_type = 'replicated'
```

Comment:
Do we need this restriction?

Reply:
Yes: https://www.moosebuild.org/job/116553/